src/sagemaker/image_uri_config/tasks.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
From the impl, 
```
task_io_schemas["inputs"]["properties"],
task_io_schemas["outputs"]["properties"],
```
we only use part of the config file. Are the rest necessary?

Reply:
No, had them there for desc. But will remove them to reduce the size of the file.

Reply:
Why this task.json is in this folder? We only have this folder for Image URIs hardcoded configs. Remove it from here.

Reply:
Thought you mentioned that this is the only folder that is onboarded with RBA.
I'll move the json file under serve/utils
Fyi @samruds

Reply:
move it to be under serve/schemas (new dir)